ee/vellum_ee/workflows/display/nodes/vellum/try_node.py

```diff
@@ -54,3 +57,43 @@ def serialize(self, display_context: WorkflowDisplayContext, **kwargs: Any) -> J
                 serialized_node_definition["name"] = node.__name__
 
         return serialized_node
+
+    @classmethod
+    def wrap(cls, error_output_id: Optional[UUID] = None) -> Callable[..., Type["BaseTryNodeDisplay"]]:
```

Comment:
the ✨ magic ✨

Reply:
neat!

[oos] We'll want to abstract this out for all adornable nodes at some point.

Reply:
yea I'm reserving that for when we're closer to implementing adornments in the UI for map/retry